src/lib/builders/dialog/tests/Dialog.spec.ts

```diff
@@ -21,6 +21,7 @@ describe('Dialog', () => {
 
 		await expect(content).not.toBeVisible();
 		await userEvent.click(trigger);
+		await sleep(1000);
```

Comment:
Ouch, is 1000 needed?

Reply:
Actually, none is needed, leftover from previous tshooting :)